### DISCUSSION TITLE
chia Node not loading

### DISCUSSION THREAD

**Discussion by 05AYAN** [Chia-Network/chia-blockchain#7090] [Support]
Chia node is not loading, tried to reboot my MacBook Pro couple of times nothing worked so far, I have also updated the chia app version to 1.1.7 and the issue didn't go still

**Comment by 05AYAN** [comment]
![Screen Shot 2021-07-01 at 8 55 59 PM](https://user-images.githubusercontent.com/86796636/124163075-44e6ae80-dab0-11eb-8608-7557bf2b4ce9.png)

**Reply by loppefaaret** [reply]
that looks very much like a corrupted blockchain database, try this:
MacOS BLOCKCHAIN DATABASE RESET

Close down Chia, and open up a terminal, and execute the below command:
```
mv ~/.chia/mainnet/db/block* ~/Desktop/
```
Then try opening up the Chia again
if it worked, you can remove the files we put on the Desktop!

this will force you to sync from 0

**Comment by 05AYAN** [comment]
It really worked. Thank you very much indeed

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
